openlibrary/catalog/merge/names.py

```diff
@@ -224,7 +224,7 @@ def match_surname(surname, name):
 
 def amazon_spaced_name(amazon, marc):
     len_amazon = len(amazon)
-    if len_amazon != 30 and len_amazon != 31:
+    if len_amazon not in {30, 31}:
         return False
```

Comment:
As `len_amazon` is only used for this comparison, how about the following:
```suggestion
    if len(amazon) not in {30, 31}:
        return False
```
I had some trouble with the deleted lines, so I am not sure if accepting this suggestion on GitHub will result in what I intend, but the suggestion as it appears is meant to be the entire body of the function.

Reply:
Nice!  I will make the change manually to ensure it does the right thing.